internal/pkg/cli/task_run.go

```diff
@@ -271,6 +286,10 @@ func (o *runTaskOpts) Ask() error {
 
 // Execute deploys and runs the task.
 func (o *runTaskOpts) Execute() error {
+	if o.dockerfilePath == "" {
+		o.dockerfilePath = defaultDockerfilePath
+	}
```

Comment:
Is there a reason why this can't be part of the default?
```go
cmd.Flags().StringVar(&vars.dockerfilePath, dockerFileFlag, defaultDockerfilePath, dockerFileFlagDescription)
```

Reply:
In the validation step we need to know whether the user has specified the --dockerfile flag. If we give a default value here instead of setting it at the execute step, then we won't be able to know whether it's simply not set or set to "Dockerfile" by the user.

I think it's possible to set the default value here while being able to tell whether the user has set it or not, by using `FlagSet.Changed("dockerfile")`, if this is preferred!

Reply:
I think I'd prefer that! So that the user sees that there is a default value for it set when the help menu is printed